packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -381,19 +403,91 @@ contract Reserve is
    * @return Returns true if the transaction succeeds.
    */
   function transferGold(address payable to, uint256 value) external returns (bool) {
```

Comment:
You could create a `transferCELO` function that just calls this one to little by little starts deprecating old gold references.